extensions/interactions/DragAndDropSortInput/directives/oppia-interactive-drag-and-drop-sort-input.component.ts

```diff
@@ -60,6 +66,24 @@ export class InteractiveDragAndDropSortInputComponent implements OnInit {
     private interactionAttributesExtractorService:
       InteractionAttributesExtractorService) {}
 
+  ngAfterViewInit(): void {
+    this.listSubscription = this.listItems.changes.subscribe((_) => {
+      this.setFocus();
+    });
+  }
+
+  ngOnDestroy(): void {
```

Comment:
This need s a frontend test.

Reply:
@Lawful2002 I have written the frontend test PTAL